apm/otel_monitor.go

```diff
@@ -243,3 +254,239 @@ func peerInfo(evt *event.CommandStartedEvent) (hostname string, port int) {
 	}
 	return hostname, port
 }
+
+func extractStatement(commandName, statement string, stripped bool) (string, error) {
+	var raw bson.Raw
+	if err := bson.UnmarshalExtJSON([]byte(statement), false, &raw); err != nil {
+		return "", nil
```

Comment:
why don't we want to return an error here? could we add a comment about this?

Reply:
My understanding is that most of the instrumentation libraries don't return errors when they fail to do the thing they're supposed to do. If they can't do it, the attribute will just be missing.
If we want to, though, we could `grip.Error` if something happens.

Reply:
Ah okay that makes sense. I think it would be nice to just add a comment to the code to explain that behavior, like the distinction between this error and the "getting elements" type errors (even if it's typical, its our library so leaving ourselves some notes doesn't hurt haha)

Reply:
Started writing the explanation out and then opted to grip.Error 10% of the time